### PR TITLE
Mark exclusion kaizen

### PR DESCRIPTION
**Pull request by roman**
@Dewdrops what do you think on these changes?

### CODE REVIEW

File: evil-visual-mark-mode.el [hunk 1]
```diff
@@ -42,7 +42,7 @@
   :prefix "evil-visual-mark-mode"
   :group 'evil)
 
-(defcustom evil-visual-mark-exclude-marks '()
+(defcustom evil-visual-mark-exclude-marks '("^" "[" "]")
```
Comment (Dewdrops) [inline evil-visual-mark-mode.el:45]: I agree string is better than characters, but we should also change the custom type from `'(repeat character)` to `'(repeat string)` .

**Comment by Dewdrops** [comment]
`^, [, ]` is just my setting for exclude mark list. I think it's a proper default. Maybe we should update the Readme.md to mention the change ?

File: evil-visual-mark-mode.el (hunk 1 above)
Comment (roman) [inline evil-visual-mark-mode.el:45]: :+1: good point, updated this

**Comment by Dewdrops** [comment]
:+1: Very quick response!

**Pull request merged**